modules/openapi-generator/src/main/resources/kotlin-client/visibility.mustache

```diff
@@ -0,0 +1 @@
+{{#nonPublicApi}}internal {{/nonPublicApi}}
```

Comment:
Question: Why put a single value into a new mustache file rather than just set a `visibility` property to be consumed in the templates? This would make sense if there were two condition (nonPublicApi=false => `public` and nonPublicApi=true => internal), but as a single value where Kotlin's default visibility is public probably makes this confusing from a user perspective.

Reply:
I followed the C# example, but I can change that 👍  Should I change it?

Reply:
Based on your other response, I think it would be best to remove `visibility.mustache` and inline this one-liner everywhere instead of the import.